skins/Seasons/index.html.tmpl

```diff
@@ -44,116 +82,16 @@
                onclick="choose_history('year')">Year</a>
           </div>
           <div id="history_day" class="plot_container">
-            <img src="daybarometer.png" alt="$obs.label.barometer"/>
-            <img src="daytempdew.png"   alt="$obs.label.outTemp" />
-            <img src="daytempfeel.png"  alt="$obs.label.feel" />
-            <img src="dayhum.png"       alt="$obs.label.outHumidity" />
-            <img src="daywind.png"      alt="$obs.label.windSpeed" />
-            <img src="daywinddir.png"   alt="$obs.label.windDir" />
-            <img src="daywindvec.png"   alt="$obs.label.windvec" />
-            <img src="dayrain.png"      alt="$obs.label.rain" />
-            #if $day.UV.has_data
-            <img src="dayuv.png"        alt="$obs.label.UV" />
-            #end if
-            #if $day.radiation.has_data
-            <img src="dayradiation.png" alt="$obs.label.radiation" />
-            #end if
-            #if $day.inTemp.has_data
-            <img src="daytempin.png"    alt="$obs.label.inTemp" />
-            #end if
-            #if $day.inHumidity.has_data
-            <img src="dayhumin.png"    alt="$obs.label.inHumidity" />
-            #end if
-            #if $day.extraTemp1.has_data or $day.extraTemp2.has_data or $day.extraTemp3.has_data
-            <img src="daytemp.png"      alt="$obs.label.extraTemp1" />
-            #end if
-            #if $day.rxCheckPercent.has_data
-            <img src="dayrx.png"        alt="$obs.label.rxCheckPercent"/>
-            #end if
+            $graphs('day', 'day')
```

Comment:
This is strange, but the first 'day' is the '$day' object, and the second is the 'day' image prefix (as in "daytempdew.png").